modules/c++/math/include/math/Round.h

```diff
@@ -68,6 +70,23 @@ template<typename T> inline T round(T value, size_t fractionalDigits)
     return (value > 0.0 ? std::floor(value * power10 + 0.5) / power10
                         : std::ceil(value * power10 - 0.5) / power10);
 }
+
+/*!
+ * Equivalent to ceil((float)numerator / denominator)
+ *
+ * \param numerator Number to divide
+ * \param denominator Non-zero number to divide by
+ * \return Result of division, rounded up
+ * \throw if denominator is 0
+ */
+inline size_t ceilingDivide(size_t numerator, size_t denominator)
```

Comment:
Since I have written the equivalent of this function roughly four thousand times, thank you!

Since you're possibly throwing, this won't get inlined anyway, so put it in the cpp (and make sure to include sys/Conf.h and except/Exception.h)

Also, update mt::ThreadPlanner to use this in its mNumElementsPerThread calculation